basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,51 @@
+"""Reads a file, edits it and saves it in a new location."""
+import os
+
+FIRST_NAME = "Keren"
+LAST_NAME = "Guez"
+
+
+def check_exists(input_file, output_folder):
+    """Checks if paths are valid and prints a message accordingly.
+
+    Args:
+        input_file (str): The path to the file that will be edited.
+        output_folder (str) : The path to the output folder.
+    Returns:
+        True : If both paths are valid.
+        False : If any path isn't valid.
+    """
+    if not os.path.exists(input_file):
+        print(f"The input file '{input_file}' doesn't exist.")
+        return False
+    if not os.path.exists(output_folder):
+        print(f"The output folder '{output_folder}' doesn't exist.")
+        return False
+    return True
+
+
+def stam(filename, output_file):
+    """Gets a file, edits it and saves it to a new location.
+
+    Args:
+        filename (str): The path to the file that will be edited.
+        output_file (str) : The path of the new file to be saved.
+    """
+    output_dir = os.path.dirname(os.path.abspath(output_file))
+    if check_exists(filename, output_dir):
```

Comment:
instead of putting all the logic under if, you can first check it, if its not exists exits the function, and if not do the logic